src/cli/flags/repo.go

```diff
@@ -17,13 +17,26 @@ const (
 
 var (
 	BackupIDFV           string
+	DeleteBackupIDFV     []string
 	AWSAccessKeyFV       string
 	AWSSecretAccessKeyFV string
 	AWSSessionTokenFV    string
 	CorsoPassphraseFV    string
 	SucceedIfExistsFV    bool
 )
 
+// AddBackupIDFlag adds the --backup flag.
+func AddDeleteBackupIDFlag(cmd *cobra.Command, require bool) {
+	cmd.Flags().StringSliceVar(
+		&DeleteBackupIDFV,
+		BackupFN, nil,
```

Comment:
re-using flag names can cause confusion and gotchas.  Since the primary difference here is to allow a list of backups, instead of a single one, I suggest a rename from `DeleteBackupIDF*` to `BackupIDsF*`.  This will allow two wholly separate flag declarations with no collision gotchas.

Reply:
Yes. Make sense. Let me update it.